package.json

```diff
@@ -36,11 +36,15 @@
 		"*.{js,html,css,md}": "prettier --write"
 	},
 	"dependencies": {
+		"body-parser": "^1.19.2",
+		"eslint-config-prettier": "^8.5.0",
```

Comment:
I think i forgot about that one everything related to eslint or prettier should be a dev dependency move that `eslint-config-prettier` in dev dependencies

Reply:
noted